modules/querier/frontend.go

```diff
@@ -0,0 +1,227 @@
+package querier
```

Comment:
this should be it's own package and module.  independent of the querier.

Reply:
Cool, will move this.